scipy/signal/filter_design.py

```diff
@@ -321,6 +321,8 @@ def freqz(b, a=1, worN=512, whole=False, plot=None, fs=2*pi):
         radians/sample (so w is from 0 to pi).
 
         .. versionadded:: 1.2.0
+    nyquist : bool, optional
+        If `nyquist` is True, `freqz` includes last frequency (Nyquist frequency) 
```

Comment:
Could maybe use a `.. versionadded:: 1.5.0` directive, but wait for Eric to review first.

Reply:
Yes it will indeed need this directive.

Also `nyquist` should probably only be used/considered in the `linspace` call when `not whole`. Otherwise in `whole` mode you'll include 0 and 2*pi, which is probably not desired. So the docs would then look something like:
```suggestion
        If `whole` is False and `worN` is an integer, setting `nyquist` to True
        will include the last frequency (Nyquist frequency).
```

Reply:
I would also reserve nyquist keyword to something else. `endpoint` or something less serious would be better.

Reply:
`include_nyquist` perhaps? Because it's really about whether or not to include the Nyquist freq. Agreed `nyquist` by itself is not great because it's not immediately clear you're not specifying the Nyquist rate or Nyquist freq of the data.

Reply:
Thank you for review. I fixed the code based on your reviews.